contracts/bancor-portal/BancorPortal.sol

```diff
@@ -105,33 +108,34 @@ contract BancorPortal is IBancorPortal, ReentrancyGuardUpgradeable, Utils, Upgra
         IBancorNetwork network,
```

Comment:
Well, I meant that we usually add 'init' to all construction arguments, but it is minor of course.